src/managers/tagManager/page/TagManager.ts

```diff
@@ -0,0 +1,52 @@
+import Log from '../../../libraries/Log';
+import { TagsObjectForApi, TagsObjectWithBoolean } from '../../../models/Tags';
+import TagUtils from '../../../utils/TagUtils';
+import { ContextInterface } from '../../../models/Context';
+import { ITagManager } from '../types';
+
+/**
+ * Manages tags for the TaggingContainer
+ */
+export default class TagManager implements ITagManager {
+    // local tags from tagging container
+    private tagsFromTaggingContainer: TagsObjectWithBoolean = {};
+    private context: ContextInterface;
+    public remoteTags: TagsObjectForApi = {};
+
+    constructor(context: ContextInterface) {
+        this.context = context;
+    }
+
+    /**
+     * @param  {boolean} isInUpdateMode
+     * @returns Promise resolving TagsObject if successful, {} if no change detected, null if failed
+     */
+    public async sendTags(_isInUpdateMode: boolean): Promise<TagsObjectForApi> {
```

Comment:
Don't see `_isInUpdateMode` being used. We should remove it if it isn't needed.

Reply:
@itrush thoughts?

Reply:
We shouldn't having have params for methods defined that are not used at all in them.
I would remove unless you are working on a fast follow up PR to start using it and it would be more work to remove and readd

Reply:
That's fine. @itrush had just mentioned she wanted to leave in for future but we can remove for now no problem